data/sql/updates/pending_db_world/rev_1638616990639666200.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1638616990639666200');
+
+ALTER TABLE `npc_trainer` ADD COLUMN `ReqSpell` INT UNSIGNED DEFAULT 0 NOT NULL AFTER `ReqLevel`; 
```

Comment:
You can drop the other file and add this to this one
```
-- Gnomish Engineering
UPDATE `npc_trainer` SET `ReqSpell`= 20219 WHERE `SpellID` IN (12897, 30575, 30574, 12907, 56473, 12905, 30570, 12903, 23096, 30568, 15633, 23129, 11454, 12906, 12759, 12902, 30569, 12899, 23489, 36955, 11454);

-- Goblin Engineering
UPDATE `npc_trainer` SET `ReqSpell`= 20222 WHERE `SpellID` IN (23486, 36954, 30565, 30566, 12755, 12718, 12908, 9273, 23078, 12717, 12716, 8895, 12758, 30563, 12760, 15628, 30560, 12754, 30558, 12715, 13240, 56514);
```

Reply:
I don't want to do it. If someone re-run that sql file, there will be errors about duplicate column.